api/src/main/java/uk/gov/dhsc/htbhf/card/model/TransferRequestDTO.java

```diff
@@ -9,17 +9,20 @@
 import lombok.Data;
 
 import java.math.BigDecimal;
+import javax.validation.constraints.NotNull;
 
 @Data
 @Builder
 @AllArgsConstructor(onConstructor_ = {@JsonCreator})
 @ApiModel(description = "Card fund transfer request object")
 public class TransferRequestDTO {
 
+    @NotNull
```

Comment:
Do we potentially want a size validator on here? Min should be £0.01 (or variable to single voucher value), max could be set so in the event of comprise we limit how much we transfer?

Reply:
Lets not worry abut validating any max, as that will be configured elsewhere (we don't want it as a constant). I'd be tempted to leave any min value out as well.